.sync.yml

```diff
@@ -13,3 +13,7 @@ docs/antora.yml:
     key: instance
     entries:
       - defaults
+
+.gitignore:
+  additionalEntries:
+    - jsonnetfile.json
```

Comment:
Side note: There's no real reason to add `jsonnetfile.json` to `.gitignore`, as there's cases where you want to commit it (additional 3rd party Jsonnet dependencies).

You can ignore it locally either in a global `.gitignore` (`git config --global core.excludesfile = /path/to/global/gitignore` and then add `jsonnetfile.json` there) or by adding it to the repo's `.git/info/exclude`.

Reply:
Ok. However if the jsonnetfile.json is in Git then the editorconfig check will fail due to missing newline at the end. If I add it manually it will fix the check, but Commodore will remove it after running tests again... or that's how I remember it.

Shall we add `insert_final_newline = false` to modulesync-control repo for `[*.*sonnet]`?